system-variables.md

```diff
@@ -1724,6 +1743,34 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
 - This variable is used to set whether the optimizer executes the optimization operation of pushing down the aggregate function to the position before Join, Projection, and UnionAll.
 - When the aggregate operation is slow in query, you can set the variable value to ON.
 
+### tidb_opt_cartesian_bcj
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: YES
+- Type: Integer
+- Default value: `1`
+- Range: `[0, 2]`
+- Indicates whether to allow broadcast CARTESIAN join. 
+- `0` means that it's not allowed, `1` means that it's allow based on `tidb_broadcast_join_threshold_count`, and `2` means it's always allowed even if the table size exceeds the threshold.
```

Comment:
Typo: "that it's allow" -> "that it's allow**ed**"

Reply:
fixed